datadog_checks_dev/datadog_checks/dev/tooling/templates/configuration/instances/http.yaml

```diff
@@ -40,6 +40,18 @@
 
     If set to `true`, this makes the check bypass any proxy
     settings enabled and attempt to reach services directly.
+- name: auth_type
+  value:
+    example: basic
+    type: string
+  description: |
+    The type of authentication to use. The available types are:
+
+      - basic
+      - digest
+      - ntlm
+      - kerberos
+      - aws
```

Comment:
We should probably add, beside each auth_type, the other configurations that need to be set respectively.